test/unittest/unittest_nntrainer_models.cpp

```diff
@@ -762,9 +762,13 @@ INSTANTIATE_TEST_CASE_P(
     mkModelTc(conv_same_padding_multi_stride, "3:1:1:10", 10),
     mkModelTc(conv_no_loss_validate, "3:1:1:10", 1),
     mkModelTc(conv_none_loss_validate, "3:1:1:10", 1)
-), [](const testing::TestParamInfo<nntrainerModelTest::ParamType>& info){
```

Comment:
I wonder if this doesn't work for you?

Reply:
I am still on Ubuntu 16.04, and libgtest is old for xenial.
I will update to 20.04 once nnstreamer build is fixed for it, and then update this.

Reply:
ah I see. Thanks